packages/fx-core/src/plugins/resource/appstudio/utils/telemetry.ts

```diff
@@ -39,6 +44,9 @@ export class TelemetryUtils {
       properties = {};
     }
     properties[TelemetryPropertyKey.component] = Constants.PLUGIN_NAME;
+    if (this.ctx.app.id) {
```

Comment:
ctx.app.id is not "remote teams app id", it the id in manifest file, and its value could be a placeholder like "{appid}". we opened a bug for that before. So please make sure you get appid from env config file.